templates/includes/install-problems.hbs

```diff
@@ -2,28 +2,42 @@
+    <h3>Bootloop of any kind or stuck boot process</h3>
+    Don't panic. Non of the commands we advised you to use can damage or brick your watch permanently.<br>
+    You can always go back to the fastboot bootloader menu, by using the <a href="https://asteroidos.org/wiki/useful-commands/#boot-to-fastboot-bootloader-menu">manual finger combos for your watch</a>.
+    <ul>
+      <li><h4>Verify that you used the correct image files</h4>Some devices like the Asus Zenwatch 2, the Fossil Gen4 models and the MTK6580 watches have more than on system image supplied, for different variants with slightly different hardware. Most watches have a graphical fastboot menu that shows the correct codename for the watch. Others need to be identified using the hardware differences noted on the install page.</li>
+      <li><h4>Clean your watch and charger contacts</h4>Even in case the fastboot flash process succeeds with no errors, it can still silently fail due to connection problems caused by dirty contacts. Clean the pins and pads with isopropyl alcohol. Ideally using that old toothbrush you keep around for electronics cleaning or a microfiber cloth. Carefully sanding the contacts (e.g. with a nailfile) can help in case of obvious corrosion.</li>
+      <li><h4>Ensure a stable USB connection</h4>Put the watch and your computer on the same flat surface. Try to not move the cable connecting your computer and the watch during the whole flashing process. In case of worn or corroded contacts and pogo-pins that lost their springiness, it might be useful to attach the watch to its cradle or charging cable using ductape or rubber bands.</li>
+      <li><h4>Repeat the flash process.</h4>Up to five tries have been reported to possibly give a successful result in the end.</li>
+    </ul>
+    <h3>Boot gets stuck even after repeated reflash</h3>
+    Congratulations, you might have found a new issue for us to solve. This should not happen when using stable builds. The nightlies are however advancing quickly and it is not always possible for us to test implications on all watches accordingly. Since the watches install pages link to the nightlies currently, you likely used those. You can try again and <a href="https://release.asteroidos.org/">install a stable build when available</a>.<br>To help us solve the problem, you could try to collect logs. It is possible that the watch does not show a UI but is already accessible using <code>ssh ceres@192.168.2.15</code> or <code>adb shell</code> in the state the boot process stops. In case ssh or ADB connections work, try the following commands to write logs to local .txt files.
+    <ul>
+      <li><code>ssh root@192.168.2.15 "dmesg" > dmesg-watch.txt</code></li>
+      <li><code>ssh root@192.168.2.15 "journalctl" > journal-watch.txt</code></li>
+      <li><code>adb shell logcat > logcat-watch.txt</code></li>
```

Comment:
The commands here for ssh and adb are confusing, since journal and logcat give different info.
on the first boot after flashing, watches will either present an ssh interface (if usb is listed as working) or an adb interface (if usb is listed as broken) - adb and ssh interfaces will never be presented simultaneously.

Also, everything in dmesg should be in journal log anyway. `adb shell logcat` also won't work on some watches since `/system/bin` isn't in path. 
I'd replace the commands with the following:
```
ssh root@192.168.2.15 "journalctl" > journal-watch.txt
ssh root@192.168.2.15 "/system/bin/logcat" > logcat-watch.txt
```
`or, if your watch presents an ADB interface, use the following commands`
```
adb shell journalctl > journal-watch.txt
adb shell "/system/bin/logcat" > logcat-watch.txt
```